diesel/src/query_builder/upsert/on_conflict_actions.rs

```diff
@@ -132,8 +152,9 @@ where
     T: Column,
 {
     fn walk_ast<'b>(&'b self, mut out: AstPass<'_, 'b, DB>) -> QueryResult<()> {
-        out.push_sql("new.");
+        out.push_sql("VALUES(");
         out.push_identifier(T::NAME)?;
+        out.push_sql(")");
```

Comment:
I liked what existed previously [as per docs](https://dev.mysql.com/doc/refman/8.0/en/insert-on-duplicate.html)

> Beginning with MySQL 8.0.19, it is possible to use an alias for the row, with, optionally, one or more of its columns to be inserted, following the VALUES or SET clause, and preceded by the AS keyword. Using the row alias new, the statement shown previously using VALUES() to access the new column values can be written in the form shown here:

```sql
INSERT INTO t1 (a,b,c) VALUES (1,2,3),(4,5,6) AS new
  ON DUPLICATE KEY UPDATE c = new.a+new.b;
```

But just went with the current approach since it was fastest 😅